jsdoc/assets/README.md

```diff
@@ -0,0 +1,116 @@
+# JSCAD V2 - Documentation
```

Comment:
Can this be cleaned up a little? For example, screen shots don't really apply here.

Reply:
Yeah.  I'd not turned an eye towards this file yet. Coming soon...